tests/cupyx_tests/scipy_tests/special_tests/test_convex_analysis.py

```diff
@@ -0,0 +1,58 @@
+import itertools
+import unittest
+
+import cupy
+import numpy
+from cupy import testing
+import scipy.special  # NOQA
```

Comment:
Separate `cupy` and `cupy.testing` imports to the bottom part (along with `import cupyx.scipy.special`)